pkg/exchange/coinbase/convert.go

```diff
@@ -20,6 +21,31 @@ func toGlobalSide(cbSide api.SideType) types.SideType {
 	return types.SideTypeNone
 }
 
+func toGlobalOrderStatus(order *api.Order) types.OrderStatus {
+	switch order.Status {
+	case api.OrderStatusRejected:
+		return types.OrderStatusRejected
+	case api.OrderStatusReceived:
+		return types.OrderStatusReceived
```

Comment:
Is Received = Update?

this means if an order is working, then you will receive incremental "ExecutedQuantity" ? you might need update the status to PartiallyFilled and add the same check like LINE35-39?

Reply:
After discussion with @kbearXD, I think received is more close to new in bbgo.
In that case, a dedicated status for received is not necessary.
I will remove it and map received status on Coinbase to new in bbgo.